content/layers.md

```diff
@@ -152,33 +152,35 @@ Source options | Description
 #### Multiple layers filter example WFS
 
 ```json
-{
-  "name": "my_custom_name",
-  "id": "name_on_server",
-  "title": "Urban",
-  "group": "Urban group",
-  "source": "mapserver_wfs",
-  "style": "urbanStyle",
-  "type": "WFS",
-  "filter": "type = 'urban'"
-},
-{
-  "name": "my_custom_name",
-  "id": "name_on_server",
-  "title": "Parklands",
-  "group": "Parklands group",
-  "source": "mapserver_wfs",
-  "style": "parklandsStyle",
-  "type": "WFS",
-  "filter": "type = 'parklands'"
-}
+[
+  {
+    "name": "my_custom_name",
+    "id": "name_on_server",
+    "title": "Urban",
+    "group": "Urban group",
+    "source": "mapserver_wfs",
+    "style": "urbanStyle",
+    "type": "WFS",
+    "filter": "type = 'urban'"
+  },
+  {
+    "name": "my_custom_name",
+    "id": "name_on_server",
+    "title": "Parklands",
+    "group": "Parklands group",
+    "source": "mapserver_wfs",
+    "style": "parklandsStyle",
+    "type": "WFS",
+    "filter": "type = 'parklands'"
+  }
+]
```

Comment:
Double underscore + suffix is missing in this example.

Reply:
Ok, the code section in the comment above ended up being a bit slimmer than I had in mind, but you'll know what I mean. I think.

Reply:
I would have gone with `my_custom_name__urban` and `my_custom_name__parklands`, but this works just fine. The message gets through. :)

Reply:
Yes, I understand, have made an update.
Could have added examples for more layer types but would be very repetitive

Reply:
> I would have gone with `my_custom_name__urban` and `my_custom_name__parklands`, but this works just fine. The message gets through. :)

Good thinking, but I don't change it now since it's the uniqueness that counts. :)